File: src/Java/myservicebus-serialization-bson/src/main/java/com/myservicebus/serialization/bson/BsonSerializationException.java
package com.myservicebus.serialization.bson;

/**
 * The MassTransit BSON envelope could not be serialized or deserialized.
 */
public final class BsonSerializationException extends RuntimeException {
    public BsonSerializationException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: src/MyServiceBus.Serialization.Bson/BsonSerializationException.cs
namespace MyServiceBus.Serialization.Bson;

/// <summary>
/// The MassTransit BSON envelope could not be serialized or deserialized.
/// </summary>
public sealed class BsonSerializationException : Exception
{
    public BsonSerializationException(string message, Exception innerException)
        : base(message, innerException)
    {
    }
}
